prow/getting_started.md

```diff
@@ -77,6 +77,14 @@ Go to that address in a web browser and verify that the "echo-test" job has a
 green check-mark next to it. At this point you have a prow cluster that is ready
 to start receiving GitHub events!
 
+As of 1.8 Kubernetes uses [Role-Based Access Control (“RBAC”)](https://kubernetes.io/docs/admin/authorization/rbac/) 
+to drive authorization decisions, allowing admins to dynamically configure policies. 
+To view the deck you might need to grant a user admin role in all namespaces for the cluster.
```

Comment:
I think we should say "To create cluster resources you need to..." instead. This is applicable to more than deck.